mempool/mempool.go

```diff
@@ -947,7 +956,6 @@ func (mp *TxPool) Count() int {
 // This function is safe for concurrent access.
 func (mp *TxPool) TxHashes() []*chainhash.Hash {
 	mp.mtx.RLock()
-	defer mp.mtx.RUnlock()
 
```

Comment:
Same thing here regarding the lock around the critical section and spacing consistency.